signing/Makefile

```diff
@@ -0,0 +1,26 @@
+include ./bbpass
```

Comment:
It might be more robust to single quote all of the arguments here - I had to fiddle with shell escapes and stuff, but it was a lot easier if the storepass and bbpass were quoted